lib/src/execution.rs

```diff
@@ -59,7 +61,7 @@ impl TxExecStrategy for EthTxExecStrategy {
             .as_mut()
             .expect("Header is not initialized");
         // Compute the spec id
-        let spec_id = block_builder.chain_spec.spec_id(header.number);
+        let spec_id = SHANGHAI;
```

Comment:
After conf change above, here is no need to hardcode.

Reply:
I will add back of this